test-it/testList.json

```diff
@@ -326,6 +326,8 @@
                 "SubVISimple.via",
                 "TemplateBasics.via",
                 "TestVI_ArrayDelete.via",
+                "TestVI_ArrayDelete_2.via",
```

Comment:
Please rename the test to provide more context as to what is being tested. 2 and 3 doesn't ring a bell.